libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -101,6 +101,7 @@ string IRGenerator::generate(ContractDefinition const& _contract)
 
 	t("CreationObject", creationObjectName(_contract));
 	t("memoryInit", memoryInit());
+	t("hasConstructor", !_contract.isLibrary());
```

Comment:
`hasConstructor` could also mean that the contract itself has a constructor - which would seem to ignore constructors of base contracts. Maybe `notLibrary`?

Reply:
That was my original idea but then I thought that it would be too specific :) I'll change it.

Reply:
Done.